src/app/containers/Ad/utilities/adSlotStyles.js

```diff
@@ -0,0 +1,91 @@
+import { css } from 'styled-components';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+
+// Ad Slot Widths for custom breakpoints
+const MPU_WIDTH_MIN = '19.75rem'; // 316px
+
+const LEADERBOARD_WIDTH_MIN = '21rem'; // 336px
```

Comment:
May I ask why have you updated these two from 300 to 316px and from 320 to 336px? How is the 1rem padding applied in the custom leaderboard breakpoint affecting these?

Reply:
That's why the update was needed - these are the minimum widths for when we start showing an ad, and before adding the extra space for the padding either side, we were rendering an ad that continued off the edge of the screen

Reply:
Good question, I will include a comment about this in the file as well. 
Because of the requirement to have `0.5rem` spacing around container at the smallest breakpoint, this forces the Ad Unit out of place and there will no longer be any vertical centering as the container is trying to force an extra `1rem` overall on the sides if we left the breakpoint as it was. 
Below are screenshots of this issue taken at 320px width if we left the smallest width for the breakpoint as the same size as the smallest size of a leaderboard as an example:

![image](https://user-images.githubusercontent.com/53564281/90384017-d9067d80-e078-11ea-9235-abb6a068da91.png)

![image](https://user-images.githubusercontent.com/53564281/90384059-e7549980-e078-11ea-82c5-ed2d8cb4b503.png)